src/platform/web/ui/session/room/timeline/BaseMessageView.js

```diff
@@ -63,7 +64,13 @@ export class BaseMessageView extends TemplateView {
                 li.removeChild(li.querySelector(".Timeline_messageSender"));
             } else if (!isContinuation && !this._isReplyPreview) {
                 const avatar = tag.a({href: vm.memberPanelLink, className: "Timeline_messageAvatar"}, [renderStaticAvatar(vm, 30)]);
-                const sender = tag.div({className: `Timeline_messageSender usercolor${vm.avatarColorNumber}`}, vm.displayName);
+                const sender = tag.div(
+                    {
+                        className: `Timeline_messageSender usercolor${vm.avatarColorNumber}`,
+                        title: vm.sender,
+                    },
+                    vm.displayName,
+                );
```

Comment:
TODO: Need to split this out as a follow-up to https://github.com/vector-im/hydrogen-web/pull/917